setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import find_packages, setup
+
+setup(
+    name="solc-select",
+    description="Manage multiple Solidity compiler versions.",
+    url="https://github.com/crytic/solc-select",
+    author="Trail of Bits",
+    version="0.1.0",
```

Comment:
This is a major change, so I suggest we make it version `*2.*`.
```suggestion
    version="0.2.0",
```